src/nl_robot_console/console.py

```diff
@@ -12,12 +12,11 @@
 
 # ----------------------------------------------------------------------------------------------------
 
-class RobotConnection:
-
+class RobotConnection(Client):
     def __init__(self, robot_name):
-        self.robot_name = robot_name
-        self.cl_robot = rospy.ServiceProxy("/state_machine/add_action", AddAction)
-        self.cl_wm = rospy.ServiceProxy(self.robot_name + "/ed/simple_query", SimpleQuery)
+        Client.__init__(self, robot_name)
+
+        self.cl_wm = rospy.ServiceProxy(robot_name + "/ed/simple_query", SimpleQuery)
```

Comment:
I guess this means "client for the world model" but please a clearer variable name